packages/kit/src/runtime/client/client.js

```diff
@@ -1038,21 +1038,31 @@ export function create_client(app, target) {
 	}
 
 	/**
-	 * @param {URL} url
+	 * @param {URL | undefined} originalUrl
```

Comment:
`originalUrl`, `rewrittenUrl`, etc. should be `original_url`, `rewritten_url`, etc.

We use snake_case for internal variables and camelCase for APIs that are exposed to users

I won't inundate you with comments everywhere that needs to be updated, but I see the same issue in other files, so please check the rest of the PR as well

Reply:
I went through the code & switched all variables to snake_case. Hopefully I didn't miss any.